docs/design-decisions/ModelPromotion.md

```diff
@@ -0,0 +1,133 @@
+# Multi-Repository Model Development and Promotion Design
+
+## Overview
+
+This documentation describes an optional design pattern for using the Model Factory.  It outlines the design of an AI Model Factory consisting of two GitHub or Azure DevOps repositories: the Data Science / Prototyping Repository and the Model Factory / MLOps Repository. This design allows data scientists to focus on model development without the constraints of adhering to strict engineering practices, while ensuring that models are rigorously tested and deployed through automated processes.
+
+## What is a Model Factory?
+
+DSToolkit v2 implements a Model Factory which enables analytics and data science teams in partnership with software engineering teams to go from model development to a stable, reliable and auditable production model artifact built and maintained with strong software engineering fundamentals such as developer experience, testing, observability, release management, DevOps and automation.
```

Comment:
Let's use something like "Model Factory is an MLOps Template for Azure ML....." or "Model Factory is a template that is based on Azure ML SDK v2....". We are not using "DSToolkit" anywhere in the repo.

Reply:
Updated, thanks!